### PR TITLE
handle prompt events with null id

### PR DESCRIPTION
**Pull request by cmarcelk**
Since wf_api_prompt_events will arrive for all audio, even audio
that our workflow didn't request, not all of those events will
have and id set in the payload. So don't assume there will be a
non-null id in the event payload.

Also fix some whitespace formatting that pycharm was complaining
about.

### CODE REVIEW

**Comment by steveims** [comment]
what can trigger a prompt event unknown to the SDK?  and is there no reason for a workflow to know about those?

**Comment by cmarcelk** [comment]
All audio playouts on the device will be accompanied by a prompt event. For example, if there is a notification tone from another workflow. In those kind of cases, I don't think the current workflow needs to know about those. I believe requests for audio playouts are serialized, so more than one at the same time will be book-ended instead of on top of each other.

File: relay/workflow.py [hunk 1]
```diff
@@ -585,16 +585,16 @@ async def handle(self, websocket):
...
                 e = self.fromJson(m)
                 self.logger.debug(f'recv: {e}')
 
                 _id = e.get('_id', None)
                 _type = e.get('_type', None)
                 request_id = e.get('request_id', None)
-    
-                if _id:
+
+                if _id in self.id_futures.keys():
```
Comment (steveims) [inline relay/workflow.py:597]: could rewrite this as:

fut = self.id_futures.pop(_id, None)
if fut:
   fut.set_result(e)

else:
   handled = False
   ...

**Review by steveims — APPROVED** [review]
lgtm. thanks!

**Pull request merged**